src/Serializer/AbstractItemNormalizer.php

```diff
@@ -517,7 +517,10 @@ protected function denormalizeRelation(string $attributeName, PropertyMetadata $
      */
     protected function getFactoryOptions(array $context): array
     {
-        $options = [];
+        $options = [
+            /* @see https://github.com/symfony/symfony/blob/v5.1.0/src/Symfony/Component/PropertyInfo/Extractor/ReflectionExtractor.php */
+            'enable_getter_setter_extraction' => true,
```

Comment:
wondering if this should be opt-in, does this break existing tests?

Reply:
No it doesn't break existing tests. However it can be considered as a BC break.